CoastalResilienceNetwork/GeositeFramework
# Update the css for recent scrollbar plugin

This fixes a bug where a new element was introduced but not managed by the
css and caused plugins not to be clickable.

Connects #466 

To test:
* Make the screen small enough that a scrollbar appears for the loaded plugins.
* Ensure you can still click on a plugin
